zerver/tests/test_user_groups.py

```diff
@@ -57,8 +57,9 @@ def test_get_direct_user_groups(self) -> None:
         othello = self.example_user("othello")
         self.create_user_group_for_test("support")
         user_groups = get_direct_user_groups(othello)
-        self.assert_length(user_groups, 1)
-        self.assertEqual(user_groups[0].name, "support")
+        self.assert_length(user_groups, 3)
+        # first two user groups are role-based system groups.
+        self.assertEqual(user_groups[2].name, "support")
```

Comment:
I don't like the assumption of ordering here; I think what we want instead is just to assert the set of names.